packages/venia-ui/lib/components/CheckoutPage/checkoutPage.css

```diff
@@ -106,9 +98,13 @@
         height: auto;
     }
 
-    .body {
-        /* Only one column in mobile view. */
-        grid-template-columns: 1fr;
-        gap: 1rem;
+    .stepper_heading {
+        padding-bottom: 1rem;
+    }
+
+    .payment_information_heading {
+        composes: stepper_heading;
+        /* On mobile, order summary has a top border, so avoid doubling up. */
+        border-bottom: unset;
```

Comment:
When you're overriding one aspect of a shorthand property, just go for the minimum necessary delta. Helps clarify the intent and in some cases (not this one) helps with transitions.

```css
border-bottom-width: 0;
```

Reply:
Fair :) idk why but I thought unset was preferred over default